src/components/drawer/drawer.component.js

```diff
@@ -16,7 +16,7 @@ import {
 } from "./drawer.style";
 import StickyFooter from "../../__internal__/sticky-footer";
 
-const SidebarContext = React.createContext();
+const DrawerSidebarContext = React.createContext();
```

Comment:
Can you add a default value to the context here as well please. I'll create a new ticket to remove the `context && context.isInSidebar` code from the consuming components as it's not relevant for you to do that here

Reply:
Update:
Adding default value causes test failures. I suggest to create a ticket for this instead of doing it in this PR.
@edleeks87 What do you think?

Reply:
Yeah that's fine we can do it all separately no problem